x-pack/test/api_integration/deployment_agnostic/services/synthetics_monitor.ts

```diff
@@ -0,0 +1,243 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { RoleCredentials } from '@kbn/ftr-common-functional-services';
+import { SYNTHETICS_API_URLS } from '@kbn/synthetics-plugin/common/constants';
+import { syntheticsMonitorType } from '@kbn/synthetics-plugin/common/types/saved_objects';
+import { EncryptedSyntheticsSavedMonitor } from '@kbn/synthetics-plugin/common/runtime_types';
+import { MonitorInspectResponse } from '@kbn/synthetics-plugin/public/apps/synthetics/state/monitor_management/api';
+import { v4 as uuidv4 } from 'uuid';
+import expect from '@kbn/expect';
+import { ProjectAPIKeyResponse } from '@kbn/synthetics-plugin/server/routes/monitor_cruds/get_api_key';
+import moment from 'moment/moment';
+import { omit } from 'lodash';
+import { KibanaSupertestProvider } from '@kbn/ftr-common-functional-services';
+import { DeploymentAgnosticFtrProviderContext } from '../ftr_provider_context';
+
+// fix saml auth type
+export class SyntheticsMonitorTestService {
+  private supertest: ReturnType<typeof KibanaSupertestProvider>;
+  private getService: DeploymentAgnosticFtrProviderContext['getService'];
+  private supertestWithoutAuth: ReturnType<typeof KibanaSupertestProvider>;
+  public apiKey: string | undefined = '';
+  public samlAuth: any;
+
+  constructor(getService: DeploymentAgnosticFtrProviderContext['getService']) {
+    this.supertest = getService('supertestWithoutAuth');
+    this.samlAuth = getService('samlAuth');
+    this.getService = getService;
+    this.supertestWithoutAuth = getService('supertestWithoutAuth');
```

Comment:
First time I've seen supertestWithoutAuth imported twice.
What's the deal?

Reply:
No deal. It's a mistake. Thanks for the catch!

Reply:
Looks like it's still imported twice?

Reply:
It looks like you're on an older commit. Here's the most recent file https://github.com/elastic/kibana/pull/198950/files#diff-f57bbeb09d4030043e0c47cac150f6d4458b2ea7d139bb99784da40b1f7488ecR26